checker/AnDO_Error.py

```diff
@@ -16,7 +25,7 @@ def __init__(self, arg):
                          names[2] + "\n" +
                          "\nIt does not respect the naming convention for "
                          "a session directory,"
-                         " which is  [yymmdd]_numéro de session (expérience)_"
+                         " which is  sess-[yymmdd]_numéro de session (expérience)_"
                          " espèce [m, o, r, s]"
                          " _ UFID animal(User friendly ID) _ "
                          "commentaire libre"
```

Comment:
let's translate this in english, and change the format... can you put something more homogeneous in terms of notations:

which is yymmdd_xxx_a_bbbb_cccc, with:
 - yymmdd is the date in the format year, month, date (6 digits, for instance 200430 for April 30, 2020)
 - xxx is the number of the session acquired on that date (3 digits, for instance 001 for the first session)
 - a is a single letter that designate the species (m for ???, o for ???, r for ???, s for ???)
 - bbbb is a string containing the user friendly identifier (UFID) of the animal
 - cccc is a string of which the use is to be decided by the research group / user (for instance to add extra info on the version of the experimental protocol, on the type of preparation etc.)

Reply:
do i  have to complete "???" in "mf for ???"

Reply:
yes, of course ;)
do you have the info? (you should, somewhere in the documents / emails / notes...)